docker-bake.hcl

```diff
@@ -58,10 +62,7 @@ target "default-args" {
         # If `ERPNEXT_VERSION` variable contains "v12" use Python 3.7. Else — 3.9.
         PYTHON_VERSION = can(regex("v12", "${ERPNEXT_VERSION}")) ? "3.7" : "3.9"
     }
-    platforms = [
-        "linux/amd64",
-        "linux/arm64"
-    ]
+    platforms = notequal("pull_request", ${GITHUB_EVENT_NAME}}, ["linux/amd64","linux/arm64"], ["linux/amd64"])
```

Comment:
What if somebody (me) wants to build images locally on `linux/arm64` platform? I don't we should bring GitHub related stuff to `docker-bake.hcl`. Let's not mess up with abstraction layers.

My suggestion is to:
- add `platform` input to `docker-build-push` resuable action
- set `platform` input in v12, v13 builds

Reference: https://github.com/docker/bake-action#inputs

Reply:
i agree this is a cleaner solution, but i'll have to put a little work into figuring it out. give me a day to see what i can come up with.

Reply:
I wound up using a variation on the above. I added an environment variable `MULTI_ARCH_BUILD` to the `docker-bake.hcl` file, with a default value of `false`.  In the `build_bench.yml` this is set to false for PRs (true otherwise).

I wasn't able to test properly on "stable" and "develop" because it requires replacing the `docker-build-push.yml` on `main`. As it stand now, multi arch can be toggled for `bench` but is disabled for `develop` and `stable`.

To allow toggling on `develop` and `stable`, `docker-build-push` on `main` has to be replaced. I have deliberately not made those changes as yet, they are relatively straightforward, but probably need to be synchronized with simultaneous changes in 'develop' and 'stable' (otherwise the build in `main` may stop working).

Reply:
have replaced the env var with an input for the `docker-build-push` action because env vars were not being accepted in the job YAML for the action.

Reply:
after numerous experiments, I finally figured out how to do what you suggested (by building against an action in a branch in my repo). that's more or less what I wound up implementing.